### ISSUE TITLE
Android 6+ permissions

### ISSUE THREAD

**Issue by kartk** [AkashBhave/LocoMoto#3]
The app won't work with current permission settings as from Android 6.0 onwords we need to ask the users whether they would give the permission or not at run time

**Comment by AkashBhave** [comment]
You are right. I will try to look into this issue as soon as possible. It shouldn't be hard considering that all I have to add is the permission request.
